app/controllers/application_controller.rb

```diff
@@ -1,5 +1,16 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) do |u|
+      u.permit(:name, :posts_counter, :email, :password, :password_confirmation)
+    end
```

Comment:
- [ ] I received the error below while signing up. I suggest you include the attribute `bio` to the permitted parameter during sign_up since it is required for sign_up as I saw on your sign_up page, and kindly remove `posts_counter` from the permitted parameter for sign_up since it is not needed during sign_up. See suggestion below;

<img width="374" alt="Code_4PP3WJrsRn" src="https://user-images.githubusercontent.com/81468675/157272051-3d7303fe-b095-4acb-8faf-664b11a2a915.png">



```suggestion
  def update_allowed_parameters
    devise_parameter_sanitizer.permit(:sign_up) do |u|
      u.permit(:name, :email, :password, :password_confirmation, :bio)
    end
```